CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.9.2
+
+v1.9.2 is a maintenance release with the following fixes and enhancements:
+
+ - Support setting principal and SASL extensions in oauth_cb
+   and handle failures (@Manicben, #1402)
+ - Wheel for macOS M1/arm64
+ - KIP-140 Admin API ACL fix:
+   When requesting multiple create_acls or delete_acls operations
+   if the provided ACL bindings or ACL binding filters are not
+   unique it throws an exception immediately instead of failing later
+   when the responses are read. (#1370).
```

Comment:
When requesting multiple create_acls or delete_acls operations,
if the provided ACL bindings or ACL binding filters are not
unique, an exception will be thrown immediately rather than later
when the responses are read.

Reply:
Done